structuredefinitions/Extension-England-RemovalReason.xml

```diff
@@ -0,0 +1,61 @@
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-England-RemovalReason" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/Extension-England-RemovalReason" />
+  <version value="1.0.0" />
+  <name value="ExtensionEnglandRemovalReason" />
+  <title value="Extension England Removal Reason" />
+  <status value="draft" />
+  <date value="2023-07-19" />
```

Comment:
Change date to 2023-06-01 (this will be changed again in near future but will help with find and replace)

Reply:
Agree. Please modify